2Dgame/Assets/Scripts/PlatformSpawnerScript.cs

```diff
@@ -123,15 +125,44 @@ void choosePlatform()
 
 	/**
 	 * Returns a random x-position to spawn a platform at.
-	 * Makes two platforms in the same position twice in a row
+	 * Makes two platforms in the same position twice
 	 * in a row less likely, though not impossible
 	 */
 	int correctX() {
-		int x = 2 * Random.Range (-3, 4);
+        int x = Random.Range(-6, 7);
```

Comment:
Why this change? This means 16 different possible x-positions instead of 8. Doesn't 8 places make the game feel more 'blocky' / '8-bity'?